### DISCUSSION TITLE
Remember the hash of a deleted image

### DISCUSSION THREAD

**Discussion by Tarnyd** [photoprism/photoprism#2237] [Ideas]
TLDR; I would love to see a category called "Previously deleted images" after an import.

I have spent some time over the past few months getting all of my scattered folders of images into PhotoPrism.
After the biggest bulk of the work was done, I started to remove some blurry images and just plain bad ones that I no longer wish to keep. Now I found another folder which contained a backup of an old phone and imported that. It turns out most of the images was already imported. This is not a problem overall as PhotoPrism does a great job of removing duplicates on import. But the problem is that I now have to go over everything again and remove the images that I want to delete since they were added again. It would be a huge time saver if PhotoPrism could have a setting for remembering the hash of images that you remove and then separate them after import so you can easily choose if you wish to add them this time or remove them again, since you have removed them once before for some reason.

**Comment by lastzero** [comment]
We had it implemented in a similar way (without selection dialog), but users asked us to change it:
- https://github.com/photoprism/photoprism/issues/300
- https://github.com/photoprism/photoprism/issues/250

Note that there could be more related issues, this is just what I found spontaneously.

**Reply by Tarnyd** [reply]
I tried to find similar cases before posting. But I missed those 🙂. Tho I believe that my proposed solution might make both sides happy. I will have a look at the other tickets in more detail later when I have more time.

**Reply by zaharcelac** [reply]
I'm in the same boat right now. 

Another approach can be:
- skip previously deleted photos by default during Import
- check "forced import" checkbox before importing, and this will ignore duplicates from "deleted" list

It might be more convenient to make your decision to "re-import deleted photos" before hitting "Import" button, than waiting until import is finished (maybe I want to shutdown my PC, my session might expire, etc.).

**Comment by xxyxxyxyx1** [comment]
Did you ever find a solution to this situation?

Similar to your story, I have located an old HDD. It contains an unsorted mix of new photos that I want to add, duplicates that will get caught correctly, and bad photos that I've already manually deleted.

**Reply by Tarnyd** [reply]
Unfortunately not. And to be honest I have not been keeping up with updates for some time, so I have no idea if this is something being considered to be implemented in any form in the future.
